### PR TITLE
fix(graphics)mobile-desktop-tweaks

### PR DESCRIPTION
**Pull request by Jesse-Box**
- Modified: Icons
- Modified: Icons

### CODE REVIEW

**Review by iker-barriocanal — APPROVED** [review]

File: svg/android.svg
Comment (cleptric) [inline svg/android.svg:1]: The color seems a bit off imo, https://developer.android.com/distribute/marketing-tools/brand-guidelines

File: svg_80x80/android.svg
Comment (cleptric) [inline svg_80x80/android.svg:1]: The color seems a bit off imo, https://developer.android.com/distribute/marketing-tools/brand-guidelines

File: svg/android.svg
Comment (Jesse-Box) [reply svg/android.svg:1]: I think that icon is out of date @cleptric 

Checkout the dotcom of android, their mascot is updated 
https://android.com/
https://en.m.wikipedia.org/wiki/File:Android_logo_2023.svg

Comment (cleptric) [reply svg/android.svg:1]: Ahh, lol. You may proceed 😄

**Pull request merged**